.editorconfig

```diff
@@ -6,20 +6,17 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 end_of_line = crlf
 
-[*.{xaml,xml,targets,props}]
+[*.{xaml,xml,targets,props,csproj}]
 indent_style = tab
+tab_width = 2
+indent_size = 2
```

Comment:
2 is different from the rest of the files. I don't really know where that comes from though but switching to 4 is likely more intersting.